core/src/test/scala/org/apache/spark/scheduler/DAGSchedulerSuite.scala

```diff
@@ -2161,6 +2161,63 @@ class DAGSchedulerSuite extends SparkFunSuite with LocalSparkContext with Timeou
     }
   }
 
+  test("After fetching failed, success of old attempt of stage should be taken as valid.") {
```

Comment:
can you rename to "After a fetch failure, success ..."
really minor, but I had to read this twice